test/tests/Artifactory/ArtifactoryDownloadClient.spec.ts

```diff
@@ -70,4 +71,14 @@ describe('Artifactory Download tests', () => {
         expect(checksum.sha1).toBeTruthy();
         expect(checksum.sha256).toBeTruthy();
     });
+
+    test('Build artifact download checksum test', async () => {
+        const dummyClient: JfrogClient = (jfrogClient = new JfrogClient({
+            artifactoryUrl: 'http://localhost:8080/artifactory',
```

Comment:
```suggestion
            artifactoryUrl: 'http://localhost:8080/artifactory',
            logger: TestUtils.createTestLogger(),
```

Prevents the ugly log message in the middle of the tests:
![image](https://user-images.githubusercontent.com/11367982/179475320-c00e8873-8913-43da-99c4-6e50ea43a4b6.png)